src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,37 @@
+<div class="footer">
+  <div class="footer-body">
+    <img src="../../../img/footer.png" class="footer-img" />
```

Comment:
```suggestion
    <img src="../../../img/footer.png" class="footer-img" alt="Switch" />
```
今回のように装飾目的でない画像に関しては、読み上げソフト使用者のことを考えるとalt属性を設定することが望ましいです。

> alt 属性を省略すると、画像がコンテンツの鍵となる部分であり、同等のテキスト表現を行うことができないことを表します。この属性に空文字列を設定すると (alt="")、この画像がコンテンツにおいて重要な箇所ではないことを示し、視覚ブラウザーではない場合は[レンダリング](https://developer.mozilla.org/ja/docs/Glossary/Rendering_engine)を省略することがあります。

https://developer.mozilla.org/ja/docs/Web/HTML/Element/img